posts/2021-02-17-multiregion-serverless-apps.md

```diff
@@ -216,17 +216,21 @@ So far we've assured ourselves that both nodes have a consistent view of the ini
 
 ```graphql
 mutation addTemp {
- add_Temperature_async(
+ add_Temperature(
    input: {
      airportCode: "EWR",
      timestamp: "2021-01-01T03:00:00Z",
      temperature: 1.5
    }
- )
+   syncMode: NODE_LEDGERED){
```

Comment:
```suggestion
   syncMode: ASYNC){
```

Reply:
Not sure if there's a reason to move to synch for this one.  Just minimizing the differences between old and new.

Reply:
I put them as `NODE_LEDGERED` out of laziness mostly. Most mutations had a `result` field that would result in a NULL value if the mutations was `ASYNC` so I left those as `NODE_LEDGERED` so the result would appear. I cleaned up the examples, and not they all should be `ASYNC`